xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/XWiki/QuickSearchUIX.xml

```diff
@@ -118,19 +118,34 @@
     </property>
     <property>
       <code>require(['jquery', 'bootstrap'], function($) {
+
+  function updateParentListState(globalSearchElement, isSearchOpen) {
```

Comment:
Looks like you don't need the `isSearchOpen` parameter since there's a `globalsearch-close` CSS class already. Also would make sense to control there the add/remove of that class here too.